x-pack/plugins/alerting/server/task_runner/transform_action_params.ts

```diff
@@ -192,10 +203,8 @@ export function transformSummaryActionParams({
     },
     alerts,
   };
-  return actionsPlugin.renderActionParameterTemplates(
-    actionTypeId,
-    actionId,
-    actionParams,
-    variables
-  );
+
+  return actionsPlugin.renderActionParameterTemplates(actionTypeId, actionId, actionParams, {
+    ...variables,
```

Comment:
curious to know why we need to spread again, just to create a shallow copy?

Reply:
I only did it to avoid TS complaining. I was waiting for this comment 🙂. The alternative is to cast. Changing the types of `renderActionParameterTemplates` will increase the scope of the PR dramatically.

Reply:
I think casting is fine here, but not a big deal either way.